lib/pbench/cli/agent/commands/generate_token.py

```diff
@@ -0,0 +1,78 @@
+"""pbench-generate-token"""
+
+import click
+import requests
+
+from pbench.agent.base import BaseCommand
+from pbench.cli.agent import pass_cli_context
+from pbench.cli.agent.options import common_options
+
+
+class GenerateToken(BaseCommand):
+    def __init__(self, context):
+        super().__init__(context)
```

Comment:
It is not clear you need this.

Reply:
I was using another of Click-based `pbench` command as a template, so I just did what that one did...and blindly followed inheritance (in both senses).  We _do_ want the `PbenchAgentContext` set up (I think...), but I suspect that you are right that the superclass is doing more than we need.  I have a work item queued to rationalize all of the command line tool infrastructure -- this should get shaken out through that.